packages/keyring/src/pair/index.ts

```diff
@@ -82,15 +81,14 @@ export default function createPair (type: KeypairType, { publicKey, secretKey }:
     },
     encodePkcs8: (passphrase?: string): Uint8Array =>
       encode({ publicKey, secretKey }, passphrase),
-    getMeta: (): KeyringPair$Meta =>
-      meta,
-    isLocked: (): boolean =>
-      (!secretKey || secretKey.length === 0),
+    get meta () {
+      return meta;
+    },
+    isLocked: (!secretKey || secretKey.length === 0),
     lock: (): void => {
       secretKey = new Uint8Array(0);
     },
-    publicKey: (): Uint8Array =>
-      publicKey,
+    publicKey,
```

Comment:
The same as above applies to the publicKey - this is set as we decode (so you could instantiate with an empty value, and unlock/decode to extract).

Reply:
Probably also incorrect...

Reply:
I think this is still an issue; verifying.

Reply:
Yes, also wanted to check that one - probably best just making it a getter anyway, that way we know. (As per my comments, when not completely awake, I do sometimes miss the plot as to when things are by value vs by ref, at least a getter won't be surprising as to intent)